pytest_mock.py

```diff
@@ -10,7 +10,7 @@
 version = '1.2'
 
 
-class MockFixture(object):
+class MockFixture:
```

Comment:
But apparently @nicoddemus knows how they work and that it does not matter in this case.
Makes the code cleaner, too - so I am +1 for keeping it like this.